recipes/prismatic/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set version = "1.2.1" %}
+
+package:
+  name: prismatic
+  version: {{ version }}
+
+source:
+  url: https://github.com/prism-em/prismatic/archive/{{ version }}.tar.gz
+  sha256: 844f659575dd98999b7c4864e850a7ec735f297ab1fb6c2d2be91d7f9720fe70 
+
+build:
+  number: 0
+  skip: True             # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}                
+    - {{ compiler('cxx') }}              
+    - {{ cdt('libxext-devel') }}         # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libxau-devel') }}          # [linux]
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxdamage') }}            # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - make                               # [unix]
+    - cmake
+  host:
+    - boost
+    - hdf5
+    - fftw
+    - qt =5.9.*
```

It should be two ``=`` or without any
```suggestion
    - qt 5.9.*
```